services/content-publishing/env.template

```diff
@@ -36,14 +36,14 @@ API_PORT=3000
 # Max file size allowed for asset upload
 FILE_UPLOAD_MAX_SIZE_IN_BYTES=2000000000
 
-# Number of seconds to keep completed asset entrie in the cache
+# Number of seconds to keep completed asset entries in the cache
 # before expiring them
 ASSET_EXPIRATION_INTERVAL_SECONDS=300
 
-# Number of seconds between content publishing batches. This is so that
-# the service waits a reasonable amount of time for additional content to publishing
-# before submitting a batch--it represents a trade-off between maximum batch fullness
-# and minimal wait time for published content.
+# Number of seconds between publishing batches.
+# This is so that the service waits a reasonable amount of time for
+# additional content to publish before submitting a batch--it represents
+# a trade-off between maximum batch fullness and minimal wait time for content
 BATCH_INTERVAL_SECONDS=12
```

Comment:
not related to documentation, but I wonder if we can eliminate this config--might make sense to just auto-calculate it based on `SECONDS_PER_BLOCK`

Reply:
Might be better for the programming, but I think keeping it in seconds makes it easier to think about for the user (dev in this case)

Reply:
What I mean is, do we even need this as an independently-configurable parameter? Or should it just be calculated to be some relationship to the block time?

Again, not relevant to this PR, but perhaps for later discussion.